styleguide/derek/examples/solutions.ejs

```diff
@@ -2,22 +2,7 @@
   <h5 class="red">Global Ecomm Pattern Library</h5>
   <h2 class="no-margin">The building blocks for your pages</h2>
   <p>We love our patterns and there are guidelines to using them so they can be as consistent and maintainable as possible. So here are the Dos and Don'ts of using our patterns</p>
-  <div class="col-md-6">
-    <h4 class="red">DO</h4>
-    <ul class="checks">
-      <li>Use patterns that coorespond to the content to go in them. ie. The features of a product should go in the features pattern</li>
-      <li>Try to fit your content in any of these very diverse patterns BEFORE utilizing the general content type</li>
-      <li>Try to use an icon from our ever growing icon library before utilizing an image if at all possible</li>
-    </ul>
-  </div>
-  <div class="col-md-6">
-    <h4 class="red">DONT</h4>
-    <ul class="times">
-      <li>bold, italicize or alter any text in any way other than centering</li>
-      <li>include buttons within any patterns other than the cta</li>
-      <li>put in any hard coded styles on any elements within patterns</li>
-    </ul>
-  </div>
+  <p>For more information on how exactly to use our guidelines please see <a href="/derek/solutions-usage">Solutions Usage</a></p>
```

Comment:
What do you mean by the examples are behind and what we have isn't great? Are there updates to the patterns that happened outside of zoolander?